experiments/dutch_auctions/dutch_auctions_stats.py

```diff
@@ -46,8 +46,65 @@ def track_stat(state, action, results):
 
 
 def auction_debt():
-    def track_stat(state, action, _results):
+    def track_stat(state, action):
         if action["key"] == "T_START":
             state["stats"]["auction_debt"] = state["dog"].Dirt
 
     return track_stat
+#net difference
+def net_diff():
+    def track_stat(state, results, action):
```

Comment:
let's have this be cumulative, just remove the if case for `"T_START"`

Reply:
ok, to still account for initialization, I did  if action["key"] == "T_START" and "difference_ratio" not in state["stats"]:
such that everything is still initialied and initialization happens once